### PR TITLE
fix issue with firehose module and update the readme files [CDS-375]

### PR DESCRIPTION
**Pull request by guyrenny**

### CODE REVIEW

File: modules/firehose/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Firehose Module
-Firehose module is designed to support CloudWatch metrics and logs.
+Firehose module is designed to support CloudWatch metrics.
```
Comment (avivgold098) [inline modules/firehose/README.md:2]: I'm not sure why not

File: README.md [hunk 2]
```diff
@@ -118,8 +118,8 @@ Run `terraform destroy` when you don't need these resources.
 - [cloudwatch-logs](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/cloudwatch-logs) - Send logs from `CloudWatch`.
 - [s3](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/s3) - Send logs from `S3` bucket.
 - [eventbridge](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/eventbridge) - Send logs from `eventbrdge`.
-- [firehose](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/firehose) - Send logs from `firehose`.
-- [vpc-flow-logs](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/vpc-flow-logs) - Send `vpc flow logs` logs from `S3` bucket.
+- [firehose](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/firehose) - Send metrics from `firehose`.
+- [vpc-flow-logs](https://github.com/coralogix/terraform-coralogix-aws/tree/master/examples/vpc-flow-logs%20) - Send `vpc flow logs` logs from `S3` bucket.
```
Comment (avivgold098) [inline README.md:122]: If we would like to correct this the naming for metrics should be cloudWatch-metric-stream for metrics, and each scenario we are supporting for logs

**Review by avivgold098 — COMMENTED** [review]
Added some advices

File: modules/firehose/README.md (hunk 1 above)
Comment (guyrenny) [reply modules/firehose/README.md:2]: The firehose integration in terraform send only metrics to coralogix and not logs

**Review by avivgold098 — APPROVED** [review]

**Pull request merged**